storage/imx6ull-flash/imx6ull-flash-test.c

```diff
@@ -499,6 +648,69 @@ void test_badblocks(void)
 }
 
 
+/* Continuously reads blocks and checks for appearing bad bits (reads few different blocks to avoid cache usage) */
+void test_read_disturb(void)
+{
+	const unsigned int blocks[] = { 1111, 3333, 5555, 7777 };        /* Tested blocks */
```

Comment:
should we first program these pages with some meaningful data with ECC?

Reply:
I've run the test a few times already and after many reads erased pages start to report `flash_uncorrectable` error too but I'm not sure how to interpret this result since as you noticed they don't have a valid ECC.
I guess the test should be run on correctly flashed pages so we can be sure of the meaning of the results.
I'll change `test_ecc()` to program tested page too.